airbyte-integrations/connectors/source-google-sheets/README.md

```diff
@@ -0,0 +1,38 @@
+# Google Sheets Source 
+
+This is the repository for the Google Sheets source connector, written in Python. 
+For information about how to use this connector within Airbyte, see [the documentation](https://docs.airbyte.io/integrations/sources/googlesheets). 
+
+## Local development
+### Build
+First, build the module by running the following from the `airbyte` project root directory: 
+```
+./gradlew :airbyte-integrations:connectors:source-google-sheets:build
+```
+
+This should generate a virtualenv for this module in `source-google-sheets/.venv`. Make sure this venv is active in your 
+development environment of choice. If you are on the terminal, run the following from the `source-google-sheets` directory: 
+```
+source .venv/bin/activate
+```
+If you are in an IDE, follow your IDE's instructions to activate the virtualenv. 
+
+**All the instructions below assume you have correctly activated the virtualenv.**. 
+
+### Unit Tests
+To run unit tests locally, from the connector root run:
+```
+python setup.py test
+``` 
+
+### Integration Tests 
+1. Configure credentials as appropriate, described below
+1. From the airbyte project root, run `./gradlew :airbyte-integrations:connectors:source-google-sheets:standardSourceTestPython`
+
+## Configure credentials
+### Configuring credentials as a community contributor
+Follow the instructions in the documentation above for generating credentials to access the Google API, then put those 
```

Comment:
these instructions are where? gitbook? if thery are elsewhere that's fine. just pointing out that in this doc there doesn't appear to be any information on how to get your google api creds.

Reply:
fixed, these are below

Reply:
oops I misunderstood what this was referring to. This is in Gitbook, will update the link.